docs/user.md

```diff
@@ -0,0 +1,193 @@
+
+##### Option A: Command Line 
+
+```shell
+oc label node <node_name> node-role.kubernetes.io/tests=""
+oc taint node <node_name> node-role.kubernetes.io/tests="":NoSchedule
+```
+
+##### Option B: Machine Set 
+
+If you have support for OpenShift's Machine API then you can create a new `MachineSet` to configure the labels and taints. See [OpenShift documentation](https://docs.openshift.com/container-platform/latest/machine_management/creating-infrastructure-machinesets.html#binding-infra-node-workloads-using-taints-tolerations_creating-infrastructure-machinesets) on how to configure a new `MachineSet`. Note that at time of certification testing, OpenShift's product documentation may not mention your infrastructure provider yet! 
+
+Here is a `MachineSet` YAML snippet on how to configure the label and taint as well:
+
+```yaml
+      metadata:
+        labels:
+          openshift-tests: "true"
+          node-role.kubernetes.io/tests: ""
+      taints:
+        - key: node-role.kubernetes.io/tests
+          effect: NoSchedule
+```
+
+
+### Privilege Requirements
+
+A user with [cluster administrator privilege](https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#creating-cluster-admin_using-rbac) must be used with the provider certification tool. You also use the default kubeadmin user if you wish. 
+
+
+## Install
+
+
+### Prebuilt Binary
+
+The provider certification tool is shipped as a single executable binary which can be downloaded from:
+
+[https://github.com/redhat-openshift-ecosystem/provider-certification-tool](https://github.com/redhat-openshift-ecosystem/provider-certification-tool)
+
+The provider certification tool can be used from any system with network access to the OpenShift cluster under test. 
+
+
+### Build from Source
+
+See the development guide for more information.
+*TODO - link*
+
+
+## Usage
+
+
+### Run provider certification tests
+
+```sh
+openshift-provider-cert run 
+openshift-provider-cert run -w # Keep watch open until completion
+```
+
+
+### Check status
+
+```sh
+openshift-provider-cert status 
+openshift-provider-cert status -w # Keep watch open until completion
+```
+
+
+### Retrieve the results
+
+The certification results must be retrieved from the OpenShift cluster under test using:
+
+```sh
+openshift-provider-cert retrieve
+openshift-provider-cert retrieve ./destination-dir/
+```
+
+You can see a summarized view of the results using:
+
+```sh
+openshift-provider-cert results retrieved-archive.tar.gz
+```
+
+
+### Environment Cleanup
+
+Once the certification process is complete and you are comfortable with destroying the test environment:
+
+```sh
+openshift-provider-cert destroy
+```
+
+You will need to destroy the OpenShift cluster under test separately. 
+
+
+### Certification Failures
+
+If you already know the reason for a certification failure then resolve the problem and re-run the provider certification tool again so a new certification archive is created. 
```

Comment:
Maybe we need to add some note like this: "We recommend running the new battery of tests in one fresh cluster (new cluster), as the e2e tests look at several metrics and logs and it could be impacted with old executions."